CITATION.cff

```diff
@@ -0,0 +1,81 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Frictionless Python Framework
```

Comment:
@roll the package is installed as `frictionless`, the repo/README title is called `frictionless-py`, the website is https://framework.frictionlessdata.io/

I suggest to call it:

```suggestion
title: 'frictionless: Python library for Data Packages'
```